extensions/wikia/ArticleComments/modules/ArticleCommentsController.class.php

```diff
@@ -25,7 +28,13 @@ public function executeIndex() {
 						$this->wg->Out->redirect( $oTitle->getLocalURL() );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)